burocrata/cli.py

```diff
@@ -78,7 +86,18 @@ def main(extension, check, verbose, directories):
             reporter.error(config_file.read_text())
             sys.exit(1)
         notice = config["tool"]["burocrata"]["notice"].split("\n")
-
+        
+        #########
+        if "exclude" in config["tool"]["burocrata"]:
+            exclude = config["tool"]["burocrata"]["exclude"]
+            exclude = pathspec.PathSpec.from_lines("gitwildmatch", exclude)
```

Comment:
Could be changed to always produce a valid `exclude` even if it's empty, like in `get_gitignore`. Then we don't need to check if `exclude` is defined when using it below.